src/Patterns/FilterPatterns.php

```diff
@@ -0,0 +1,200 @@
+<?php
+/**
+ * Title: Pattern example
+ * Slug: woocommerce-blocks/pattern-example
+ * Categories: WooCommerce
+ * Block Types: core/paragraph
+ */
```

Comment:
Why is this comment block here?

Reply:
Oops, I copy pasted it to try something and I forgot to remove it 😅